src/cadenza.js

```diff
@@ -844,19 +844,54 @@ export class CadenzaClient {
     return this.#fetch(resolvePath(source), params, signal);
   }
 
+  /**
+   * Fetch object info from a workbook view.
+   *
+   * @param {EmbeddingTargetId} source - The workbook view to fetch object info from.
+   *   Currently only map views are supported.
+   * @param {(WorkbookLayerPath | string)[]} [layerPath] - Layer path to identify the layer
+   *  (identified using layer paths or print names)
+   * @param {unknown[][]} objectIds - The IDs of the objects to select
+   * @param {object} [options] - Options
+   * @param {FilterVariables} [options.filter] - Filter variables
+   * @param {AbortSignal} [options.signal] - A signal to abort the data fetching
+   * @param {Boolean} [options.useMapSrs] - Use the map SRS instead of WGS84
+   * @param {Boolean} [options.fullGeometries] - Return non-simplified geometries
+   * @return {Promise<Response>} A `Promise` for the fetch response
+   * @throws For invalid arguments
+   */
+  fetchObjectInfo(source, layerPath, objectIds, { filter, signal, useMapSrs, fullGeometries } = {}) {
+    this.#log('CadenzaClient#fetchObjectInfo', ...arguments);
+    const params = createParams({
+      filter});
+    return this.#fetch(
+      resolvePath(source) + "/objectinfo" ,
```

Comment:
Please use the development watch (npm start) when developing Cadenza JS. The CI pipeline will not pass with code formatting issues.